src/styles/stars.css

```
@@ -0,0 +1,18 @@
+.stars_star {
+  background: url(/images/star.svg) no-repeat center;
+  width: 16px;
+  height: 16px;
+  display: flex;
+  margin-right: 4px;
+}
+
+.stars .stars_star:nth-child(-n+4) {
+  background-image: url(/images/star-active.svg);
+}
```

```suggestion

.stars--4 .stars_star:nth-child(-n+4) {
  background-image: url(/images/star-active.svg);
}
```

add styles for other modifiers